### DISCUSSION TITLE
Executable exploded in size

### DISCUSSION THREAD

**Discussion by thany** [NickeManarin/ScreenToGif#1186] [Q&A] [answered]
I have STG 2.34.1 portable, which is about 5MB.

This new version 2.38 is 175MB, and even the "light" version is 21MB. What gives? What could be so important that has to be included, that isn't included in 2.34, to weigh in so heavy?

Also the requirement for .NET 6 is ridiculous. Version 5 is the standard installed on most pc's. But even with such a bulky framework around, it still needs over 4 times as big an executable? (no, I wouldn't even consider the non-light version, that one is just mental)

**Comment by thany** [comment]
I guess [this issue](https://github.com/NickeManarin/ScreenToGif/issues/579) is what got you (author) started on moving this app to .NET 6?

Bad move, innit. You've seen all the issues around this.

What would it take to move back to 5, and presumably get the file size down to normal proportions again?

**Comment by NickeManarin** [comment, marked as answer]
I don't like your tone.
This is not a good critic, it's just a rant.

I spent years working on this app and you are able to use it for free.
Nobody is making you use it or update it.
